web/dataselectie/datasets/brk/tests/fixture_utils.py

```diff
@@ -298,6 +299,36 @@ def create_eigendom1():
         )
     ]
 
+def create_eigendommen(number_of_instances: int) -> list:
+    """
+    depends on kadastraal object and categroie fixtures
+    args:
+        number_of_instances: number of objects to create
+    :return: a list of eigendom objects
+    """
+    number_of_instances = number_of_instances + 1
+    create_eigenaar_categorie()
+    kadastraal_objects = [create_kadastraal_object for _ in range(1,number_of_instances) ]
```

Comment:
Small thing, but spacing in `range(1, number_of_instances)]` is a bit off.

Reply:
Check :+1: